src/rocksdb.rs

```diff
@@ -513,6 +513,71 @@ impl DB {
         self.get_cf_opt(cf, key, &ReadOptions::new())
     }
 
+    pub fn multi_get_cf_opt(&self,
```

Comment:
I think for easy use, we can declare the return type as `Result<Vec<Option<DBVector>>, String>`, If we meet any error in multi_get, we will return error directly. 

/cc @BusyJay @andelf @zhangjinpeng1987